### PR TITLE
Cards from System Gateway spoiler season

### PR DESCRIPTION
**Pull request by mxmxyz**
Some new HB cards and some new Criminal cards, respectively from [here](https://kitchentablenetrunner.podbean.com/e/episode-22-getting-better-and-gateway-scoops/) and [here](https://www.youtube.com/watch?v=4YsBncJQvVQ).

First time contributing to the repo so tell me if I did something wrong.

### CODE REVIEW

**Comment by gereons** [comment]
Thanks for your contribution! Please make sure to run the included `validate.py` script when submitting a PR, this check is currently failing and must be "green" (ie, report 0 formatting and validation errors) for us to accept the PR.

These are the errors currently reported:

```
Validation error in card: (pack code: 'sg' card code: '30040' title: 'Seamless Launch')
u'operation' is not one of [u'asset', u'upgrade']

Validation error in card: (pack code: 'sg' card code: '30041' title: 'Sprint')
u'operation' is not one of [u'asset', u'upgrade']
```

File: pack/sg.json [hunk 1]
```diff
@@ -41,6 +41,67 @@
...
+        "flavor": "\"Braggarts chase big heists. Patience enriches skimming fractions of a credit at a time.\" -Zahya",
+        "illustrator": "Martin de Diego Sādaba",
+        "keywords": "Console",
+        "pack_code": "sg",
+        "position": 14,
+        "quantity": 3,
+        "side_code": "runner",
+        "stripped_text": "+1 mu Whenever you make a successful run, place 1 credit on this hardware. Click: Place 1 credit on this hardware, then take all credits from it. Limit 1 console per player.",
+        "stripped_title": "Pennyshaver",
+        "text": "+1[mu]\nWhenever you make a successful run, place 1[credit] on this hardware.\n[click]: Place 1[credit] on this hardware, then take all credits from it.\nLimit 1 console per player.",
```
Comment (gereons) [inline pack/sg.json:59]: "console" should be in a \<strong> tag

File: pack/sg.json [hunk 2]
```diff
@@ -142,6 +224,82 @@
...
+        "illustrator": "David Lei",
+        "pack_code": "sg",
+        "position": 40,
+        "quantity": 3,
+        "side_code": "corp",
+        "stripped_text": "Place 2 advancement counters on 1 installed card that you did not install this turn.",
+        "stripped_title": "Seamless Launch",
+        "text": "Place 2 advancement counters on 1 installed card that you did not install this turn.",
+        "title": "Seamless Launch",
+        "type_code": "operation",
```
Comment (gereons) [inline pack/sg.json:262]: This can't be right

File: pack/sg.json [hunk 3]
```diff
@@ -142,6 +224,82 @@
...
+        "illustrator": "Galen Dara",
+        "pack_code": "sg",
+        "position": 41,
+        "quantity": 3,
+        "side_code": "corp",
+        "stripped_text": "Draw 3 cards. Shuffle 2 cards from HQ into R&D.",
+        "stripped_title": "Sprint",
+        "text": "Draw 3 cards. Shuffle 2 cards from HQ into R&D.",
+        "title": "Sprint",
+        "type_code": "operation",
```
Comment (gereons) [inline pack/sg.json:280]: This can't be right

**Review by gereons — CHANGES_REQUESTED** [review]

**Comment by plural** [comment]
hey there!  Thanks for the patch!

We (NISEI web dev team) are actually going to publish everything that was spoiled during community week on Sunday.  We'll wrap things up on the 28th with the final cards. Keep an eye on the repo past this month though, and reach out if you see any other patches you wanna contribute.

**Comment by mxmxyz** [comment]
I will, thank you for your work ^^

**Comment by gereons** [comment]
Closing this in favor of #562

**Pull request closed without merging**